glossary.adoc

```diff
@@ -0,0 +1,95 @@
+[[glossary]]
+== Glossary
+
+|===
+
+| AIA | Advanced interrupt architecture (AIA) is an architecture for handling
+interrupts.
+
+| ABI | Application binary interface (ABI).
+
+| AP | Application processors (AP)s can support commodity operating systems,
+ hypervisors/VMMs and applications software workloads. The AP subsystem
+ may contain several processing units, on-chip caches, and other controllers
+for interfacing with memory, accelerators, and other fixed-function logic.
+Multiple APs may be used within a logical system.
+
+| Attestation | The process by which a relying party can assess the
+security posture of the confidential workload based on verifying a set of
+HW-rooted cryptographically-protected evidence.
+
+| Confidential Computing | A computing paradigm that protects data in use by
+performing computation in a hardware-based, attested TEE.
+
+| CoVE | Confidential VM extension (CoVE) is the set of RISC-V ABI extensions
+defined in this specification that enables confidential computing on RISC-V
+platforms. In some deployment models, the CoVE ABI leverages the RISC-V ISA
+extensions specified in the RISC-V Supervisor Domains specification.
+CoVE is a Trusted Execution Environment (TEE) ABI for Application Processors.
+A supervisor domain that provides HW-isolation for workload data assets when in
```

Comment:
Was it intentional to equate a supervisor domain to a CoVE.

Reply:
no - reused some of the glossary terms :) - will fix